typed-racket-lib/typed-racket/rep/object-rep.rkt

```diff
@@ -16,6 +16,7 @@
 (def-pathelem StructPE ([t Type?] [idx natural-number/c])
   [#:frees (λ (f) (f t))]
   [#:fold-rhs (*StructPE (type-rec-id t) idx)])
+(def-pathelem FieldPE () [#:fold-rhs #:base])
 
```

Comment:
Can you let-alias an expression with this sort of path? ex: (let ([x exp]) ...) where the object of exp has a FieldPE in its path?

If so, I think a change similar to that in update would need to be added to 
https://github.com/takikawa/typed-racket/blob/priv-field-occ-type/typed-racket-lib/typed-racket/types/path-type.rkt